packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -1420,6 +1426,12 @@ export class ResourceType {
   public static readonly SAGEMAKER_MODEL = new ResourceType('AWS::SageMaker::Model');
   /** Amazon SageMaker notebook instance */
   public static readonly SAGEMAKER_NOTEBOOK_INSTANCE = new ResourceType('AWS::SageMaker::NotebookInstance');
+  /** Amazon SageMaker workteam */
+  public static readonly SAGEMAKER_WORKTEAM = new ResourceType('AWS::SageMaker::Workteam');
+  /** Amazon SES Configuration Set */
+  public static readonly SES_CONFIGURATION_SET = new ResourceType('AWS::SES::ConfigurationSet');
```

Comment:
This one is not in the list of `Newly supported resource types`, but I see it present in https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#amazonsimpleemailservice 👍